docs/graphcache/errors.md

```diff
@@ -395,3 +395,17 @@ This error occurs when you are using an `interface` or `union` rather than an
 implemented type for these.
 
 Check the type mentioned and change it to one of the specific types.
+
+## (27) Invalid Cache write
```

Comment:
Should we also add some notes in the Local Resolvers section of the graphcache docs? That is where I want to initially find if this was an invariant

Reply:
That's a good point 👍